.github/workflows/ruby.yml

```diff
@@ -0,0 +1,50 @@
+name: Ruby
+
+on: [push]
```

Comment:
@eitoball you can drop this file if you want and just rely on circle for now. Github actions can't take AWS keys from forked PRs, so we'll have to re-vamp the build/deploy model to use github artifacts before this is gonna work very nicely.